samples/unity-of-bugs/Assembly-CSharp.csproj

```diff
@@ -0,0 +1,647 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <LangVersion>latest</LangVersion>
```

Comment:
Do we actually need these two `csproj`?

I'm able to open unity-of-bugs on Unity and edit the scripts without the need of these.

The original repo doesn't have it: https://github.com/bruno-garcia/unity-of-bugs

Reply:
Unity generates it. I'll check, nonetheless.